src/engine/server/server.cpp

```diff
@@ -1915,5 +1902,5 @@ void CServer::SendInitialData(int ClientID)
 
 bool CServer::GetClientProtocolCompatibility(int ClientID, int Protocol) const
 {
-	return (m_aClients[ClientID].m_Protocol == MODAPI_CLIENTPROTOCOL_TW07MODAPI);
+	return (m_aClients[ClientID].m_Protocol == Protocol);
```

Comment:
But then, the function would really not match it's purpose :open_mouth: And the `int Protocol` argument will be completely useless?
I thought this is the purpose of the function no?
`GetClientProtocolCompatibility(1, MODAPI_CLIENTPROTOCOL_TW07MODAPI);` would check if the protocol is compatible with GUI, Sprites, ...